src/components/common/SelectorButton/index.tsx

```diff
@@ -3,46 +3,45 @@ import { useState } from 'react'
 
 import { palette } from '@/styles/palette'
 
-type ToggleButtonProps = {
+type SelectorButtonProps = {
   buttonName: string
   selectedButtonColor: string
-  defaultButtonColor?: string
-  selectedTextColor: string
-  onClick?: () => void
+  defaultButtonColor: string
+  textColor: string
+  onClick?: (selected: boolean) => void
+  selected?: boolean
```

Comment:
p4: isButtonSelected가 좋지 않을까요?
```suggestion
  isButtonSelected?: boolean
```